delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/Supervisor.scala

```diff
@@ -356,7 +360,13 @@ object Supervisor {
                               _      <- Task.delay(log.info(s"Destroying '${metadata.module}/${metadata.name}'..."))
                               _      <- stopProjection(s)
                               _      <- Task.when(s.executionStrategy == PersistentSingleNode)(projections.delete(name))
-                              _      <- clear
+                              _      <- onDestroy
+                                          .retryingOnSomeErrors(
+                                            retryStrategy.retryWhen,
+                                            retryStrategy.policy,
+                                            retryStrategy.onError
+                                          )
```

Comment:
Can we make it so that this method just takes a retry strategy? Would be much more readable

Reply:
I created an issue to introduce this method and replace it everywhere #3928